modules/appeals_api/spec/sidekiq/appeal_received_job_spec.rb

```diff
@@ -4,59 +4,221 @@
+            expect(vanotify_client).to have_received(:send_email).with(
+              {
+                date_submitted: expected_date_submitted,
+                email_address: appeal.claimant.email,
+                personalisation: { first_name: appeal.claimant.first_name, veterans_name: appeal.veteran.first_name },
+                template_id: claimant_sc_template_id
+              }
+            )
+          end
+        end
       end
     end
 
-    it 'uses sc email template' do
-      with_settings(Settings.vanotify.services.lighthouse.template_id,
-                    supplemental_claim_received: 'sc_veteran_template') do
-        opts.merge!('receipt_event' => 'sc_received')
-        expect { job.perform(opts) }
-          .to trigger_statsd_increment('api.appeals.received.claimant.email.sent',
-                                       tags: ['appeal_type:sc', 'claimant_type:veteran'], times: 1)
-        expect(client).to have_received(:send_email).with(hash_including(template_id: 'sc_veteran_template'))
+    describe 'errors' do
+      let(:expected_error) { nil }
+
+      before do
+        allow(FeatureFlipper).to receive(:send_email?).and_return(true)
+        allow(VaNotify::Service).to receive(:new).and_return(vanotify_client)
+        expect do
+          with_settings(*settings_args) { job.perform(appeal_id, appeal_class_str, date_submitted_str) }
+        end.to raise_error expected_error
+      end
+
+      context 'appeal PII not available' do
+        let(:expected_error) { /Missing PII for #{appeal.class.name} #{appeal_id}/ }
+        let(:appeal) do
+          hlr = create(:higher_level_review_v2)
+          hlr.update!(form_data: nil, auth_headers: nil)
+          hlr
+        end
+
+        it 'does not send email' do
+          expect(vanotify_client).not_to receive(:send_email)
+        end
+      end
+
+      context 'appeal with given appeal_id not found' do
+        let(:appeal_id) { SecureRandom.uuid }
+        let(:expected_error) { /find #{appeal_class_str}.*#{appeal_id}/ }
+
+        it 'does not send email' do
+          expect(vanotify_client).not_to receive(:send_email)
+        end
+      end
+
+      context 'submitted_date_str with incorrect format' do
+        let(:expected_error) { /iso8601/ }
+        let(:date_submitted_str) { 'not-a-date' }
+
+        it 'does not send email' do
+          expect(vanotify_client).not_to receive(:send_email)
+        end
+      end
+
+      context 'missing settings for VANotify templates' do
+        let(:expected_error) { /template.*#{hlr_template_name}/ }
+        let(:settings_args) { [Settings.vanotify.services.lighthouse.template_id, {}] }
+
+        it 'does not send email' do
+          expect(vanotify_client).not_to receive(:send_email)
+        end
       end
     end
   end
 
-  describe 'va notify claimant email templates' do
+  describe 'va notify claimant email templates', skip: 'skipped to avoid changing too many lines in a single PR' do
```

Comment:
💀 I tried getting rid of these specs but it put me over the 500 line limit. I couldn't think of a way to break this down further so I just skipped these for now - I can delete them in a follow-up PR.